### PR TITLE
WIP - 142655867 drafting functionality portfolio

### PR DESCRIPTION
**Pull request by drod1000**
@mollybrown @jdconrad89
This PR will require to run `rake db:migrate` and `rake db:test:prepare`

FYI @neight-allen

### CODE REVIEW

File: app/assets/javascripts/components/portfolio/_portfolio_form.js.jsx [hunk 1]
```diff
@@ -89,8 +109,13 @@ var PortfolioForm = React.createClass({
...
-            ? <button className="btn waves-effect waves-light portfolio-btns" type="submit" name="action" value="Save Profile" onClick={this.handleInsert}>Save Profile</button>
-            : <button className="btn waves-effect waves-light portfolio-btns" type="submit" name="action" value="Save Profile" onClick={this.handleUpdate}>Save Profile</button>
+            ?
+            <div>
+              <button className="btn waves-effect waves-light portfolio-btns" type="submit" name="action" value="Save Draft" onClick={this.handleDraftInsert}>Save as Draft</button>
+              <button className="btn waves-effect waves-light portfolio-btns" type="submit" name="action" value="Save Profile" onClick={this.handleInsert}>Save Profile</button>
+            </div>
+            :
+            <button className="btn waves-effect waves-light portfolio-btns" type="submit" name="action" value="Save Profile" onClick={this.handleUpdate}>Save Profile</button>
           }
```
Comment (neight-allen) [inline app/assets/javascripts/components/portfolio/_portfolio_form.js.jsx]: Let's rename "Save Profile" to "Publish Profile" so there's no confusion.

**Review by neight-allen — DISMISSED** [review]
Nice work. One small wording change.

File: app/assets/javascripts/components/portfolio/_portfolio_form.js.jsx (hunk 1 above)
Comment (Laszlo-JFLMTCO) [reply app/assets/javascripts/components/portfolio/_portfolio_form.js.jsx]: @neight-allen I added the requested change to this branch.

FYI @drod1000 @mollybrown @jdconrad89

**Comment by drod1000** [comment]
Reviewed with @Laszlo-JFLMTCO @jdconrad89 @mollybrown

**Pull request merged**